draft-ietf-quic-multipath.md

```diff
@@ -885,6 +884,37 @@ two or more active paths during the connection lifetime. Different applications
 Once the implementation has decided which paths to keep alive, it can do so by sending Ping frames
 on each of these paths before the idle timeout expires.
 
+## Connection ID Changes and NAT Rebindings
+
+{{Section 5.1.2 of QUIC-TRANSPORT}} indicates that an endpoint
+can change the Connection ID it uses for to another available one
+at any time during the connection. As such a sole change of the Connection
+ID without any change in the address does not indicate a path change and
+the endpoint will keep the same congestion control and RTT measurement state.
+
+While endpoints assign a Connection ID to a specific sending 4-tuple,
+networks events such as NAT rebinding may make the packet's receiver
+observe a different 4-tuple. Servers observing a 4-tuple change will
+performs path validation (see {{Section 9 of QUIC-TRANSPORT}}).
+If the path validation process succeeds, the endpoints need to reset
+their congestion controller and round-trip time estimator for the new
+path as specficied in  {{Section 9.4 of QUIC-TRANSPORT}}.
```

Comment:
Why not simply refers to that section directly, that also states that servers may keep their current state if they suspect the event to be a NAT rebinding?
```suggestion
If the path validation process succeeds, the endpoint
follows considerations of {{Section 9.4 of QUIC-TRANSPORT}}.
```

Reply:
I would prefer to keep this because I find it annoying to provide a vague reference without giving an indication what this reference is for. Why do you think the current text is not good?

Reply:
Because it somewhat contradicts what Section 9.4 states.

> On confirming a peer's ownership of its new address, an endpoint MUST immediately reset the congestion controller and round-trip time estimator for the new path to initial values (see Appendices [A.3](https://www.rfc-editor.org/rfc/rfc9002#appendix-A.3) and [B.3](https://www.rfc-editor.org/rfc/rfc9002#appendix-B.3) of [[QUIC-RECOVERY](https://www.rfc-editor.org/rfc/rfc9000.html#QUIC-RECOVERY)]) unless the only change in the peer's address is its port number. Because port-only changes are commonly the result of NAT rebinding or other middlebox activity, the endpoint MAY instead retain its congestion control state and round-trip estimate in those cases instead of reverting to initial values.

We could just adapt the text to be like "the endpoint updates the path's congestion controller and round-trip time estimator according to {{Section 9.4 of QUIC-TRANSPORT}}".

Reply:
```suggestion
If path validation process succeeds, the endpoints set
the path's congestion controller and round-trip time
estimator according to {{Section 9.4 of QUIC-TRANSPORT}}.
```